tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -1468,4 +1468,12 @@ public function getGeneratesFloatDeclarationSQL()
             array(array('precision' => 8, 'scale' => 2), 'DOUBLE PRECISION'),
         );
     }
+
+    public function testItEscapesStringsForLikeProperly() : void
```

Comment:
The "properly" part seems redundant. It's implied that the test will only pass if the API is implemented properly.